tox.ini

```diff
@@ -0,0 +1,9 @@
+[tox]
```

Comment:
Could you add a note here that this is used for local development + refer to the readme?

Reply:
Will do.